src/components/05-text-input/text-input.hbs

```diff
@@ -0,0 +1,5 @@
+<label for="demo-1">Text Input</label>
```

Comment:
Let's make this a prop you can pass to the `context` object instead of hard coding an `id`. That way we can use more than one per page with unique label/input pairs.